docs/Deployment_guide.rst

```diff
@@ -0,0 +1,90 @@
+************************************
+Document to guide user to do changes for deployment of kubernetes feature in FIWARE Lab GUI
+************************************
+ 
+Fiware lab User can deploy Fiware GEs over kubernetes cluster by selecting the GEs from the catalogue provided in GUI. To access UI for deploying Fiware GE from horizon dashboard the user clicks on the kubernetes button in the horizon dashboard, they will get redirected to the UI Dashboard page where user will get options to create and manage kubernetes cluster and deploy Fiware GEs.
+To directky access the UI, the changes need to be made on the Horizon Dashboard code. This guide will show the necessary required changes. 
+
+Introduction:
+----------------------------------------------
+This Document provide step by step guide user to do changes in code of FIWARE Lab GUI and provide a button in Overview tab of FIWARE LAB GUI to redirect to kubernetes support GUI.
+
+**Step 1:** For deployment we have to do change in code so that we can get Project ID or Tenant ID and also Instance IP for all the instances on which user want to deploy kubernetes cluster.

```

Comment:
Change to: **Step 1:**  For deployment we have to do change in Horizon code so that we can get Project ID and Instance IP for all the instances on which user want to deploy kubernetes cluster.

Reply:
Updated